synapseclient/extensions/curator/schema_generation.py

```diff
@@ -2855,7 +2767,7 @@ def define_data_model_relationships(self) -> dict:
                 "required_header": False,
                 "edge_rel": False,
                 "node_attr_dict": {"default": None},
-                "allowed_values": [enum.value for enum in JSONSchemaType],
+                "allowed_values": ["string", "number", "integer", "boolean"],
```

Comment:
@andrewelamb Here I hard-coded the allowed values since JOSNSchemaType is no longer there.

Reply:
But that's what the JSONSchemaType was for. I don't think it should be removed.